hmcts/pre-portal
# S28 3432: Browse Page Status Column

### JIRA ticket(s)
- S28-3432


### Change description
- Add "Status" column to browse page table




> [!IMPORTANT]
> This PR requires manual testing by QA. Please notify the QA team @hmcts/pre-rec-evidence-qa.

**QA instructions**
- Make sure you have recordings shared, some with the case pending closure and some open testing.
- On browse page see new column called `Status`
- On recordings for open case, see status `Active`
- On recordings for pending closure case, see status `Pending Closure` and some information about when the case is expected to close. 





> [!CAUTION]
> Merging this pr will make changes visible in production
